File: Apps/iOS7/HelloiPhone/HelloiPhone/ViewController.m
//
//  ViewController.m
//  HelloiPhone
//
//  Created by Clemens Wagner on 12.07.13.
//  Copyright (c) 2013 Cocoaneheads. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)viewWillAppear:(BOOL)inAnimated {
    [super viewWillAppear:inAnimated];
    NSLog(@"[+] viewWillAppear:");
    [self.label setText:@"Moin!"];
}

- (IBAction)go:(id)sender {
    NSLog(@"[+] go:");
    NSError *theError = nil;
    NSURL *theURL = [NSURL URLWithString:@"http://www.rodewig.de/ip.php"];
    NSString *theIP = [NSString stringWithContentsOfURL:theURL encoding:NSASCIIStringEncoding error:&theError];

    if(theError == nil) {
        NSLog(@"[+] IP: %@", theIP);
    }
    else {
        NSLog(@"[+] Error: %@", [theError localizedDescription]);
    }
    [self.label setText:theIP];
}

@end

File: Apps/iOS8/Swift/Hello Swift/Hello Swift/ViewController.swift
//
//  ViewController.swift
//  Hello Swift
//
//  Created by Klaus Rodewig on 30.09.14.
//  Copyright (c) 2014 Cocaneheads. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var label: UILabel!
    @IBAction func go(sender: AnyObject) {
        let theURL = NSURL(string: "http://www.rodewig.de/ip.php")
        var theIP = String(contentsOfURL: theURL!, encoding: NSUTF8StringEncoding, error: nil)
        label.text = theIP
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        label.text = "Moin"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

